test/OWNERS

```diff
@@ -1,7 +1,10 @@
 approvers:
-  - Ark-kun
-  - IronPan
-  - chensun
```

Comment:
Are you sure?

Reply:
IronPan and I are OWNERS at root folder, so not really necessary to repeat us here.